init.rb

```diff
@@ -20,6 +21,8 @@
     Rails.application.config.after_initialize do
       # this is to expose helper methods to the hook class as they are used inside the controller hooks
       Redmine::Hook::Helper.include AdvancedMessengerHelper
+      IssuesController.send(:include, AdvancedMessengerIssuesControllerPatch)
```

Comment:
It was because the two patches contains some common code and some other some other distinct code. I changed this in order to keep only one include in the init.rb, and include the base code in each specific patch:
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/02136d11-c24a-4d26-9aac-2e9ea3aeeb46)